master/docs/manual/cfg-reporters.rst

```diff
@@ -1012,6 +1012,48 @@ As a result, we recommend you use https in your base_url rather than http.
     :param boolean verify: disable ssl verification for the case you use temporary self signed certificates
     :param boolean debug: logs every requests and their response
 
+.. bb:reporter:: StashPRCommentPush
+
+StashPRCommentPush
```

Comment:
I agree it would be a good idea to rename it as BitbucketServer, to distinguish it from the existing Bitbucket (cloud) APIs.

Do you have a preferred approach for keeping the old name available?